faq.html

```diff
@@ -155,10 +155,13 @@ <h2 id="general">General</h2>
       <a href="{{page.root}}/img/software-carpentry-banner.png">Software Carpentry logo</a>
       are both trademarked,
       and may only be used with our explicit prior permission.
-      We normally grant this to any training course that covers our
-      <a href="#core-topics">core topics</a>
-      has at least one <a href="{{page.root}}/pages/team.html">certified instructor</a> teaching,
-      and sends us summary information about attendees.
+      We normally grant this to any training course that:
```

Comment:
> do we *need* to have a homepage before something can be a SWC-branded event?

No. What we need is that the workshop has one homepage that we can get the metadata.

@gvwilson and @wking Can we replace https://github.com/swcarpentry/site/blob/master/bin/preprocess.py and https://github.com/swcarpentry/site/blob/master/config/workshops.yml with queries to `amy`? If this is the case, I'm +1 to leave the host has the workshop homepage as s/he prefer.